### DISCUSSION TITLE
[Kavita/Backend] Comic Book Reviews

### DISCUSSION THREAD

**Discussion by Cducharme84** [Kareadita/Kavita#2781] [Ideas]
### Idea Description

[General]
As we know, there's not a great site for scraping Comic Reviews for Kavita's use/display and certainly not one to voluntarily send user reviews back to. I propose something similar to the project linked here https://nycdatascience.com/blog/student-works/scraping-comic-book-reviews-critics-vs-fans/, where they were able to scrape a major data source and receive what I'll call the "starter kit" of issue and series reviews.

I propose using either the schema show in that link or one created by Kavita development but populated initially with that data and maybe one or two other publicly available repositories if able. Once populated an enabling of voluntary user synced reviews could happen to allow users to input the review into their Kavita and the review can be collected at clear interval (whether daily or more of course is up for discussion).

Data needed to keep issue/series straight per review:
Series Name
Issue number
Cover date
Publisher
Volume

Optional:
Store Date
CV/Metron ID - I do recommend support for both but since Kavita after merging foundational changes will already be parsing comicinfo.xml appropriately it should be trivial for Kavita to record issue ID.

Most of the above is already collected by Kavita for use in its DB. 

Biggest barrier I see:
Backend management and support - There will be some lift either by Joe or the community as a whole in maintaining and hosting a new data source, so it will likely make sense to place behind the K+ service to drive new Western Comic oriented subscribership to help defer any financial impact. Though a relatively cheap host and server would work pretty well with the  restriction to K+ members.

Lack of data:
Despite the existence of a few major data sources we lack many community driven review sources that are easily accessed by Kavita's backend this will necessitate the solicitation/promotion of users reviewing and turning on sharing. Unfortunately the best solution to that (wider exposure) would mean keeping it available to all users and just hope the k+ subscriptions to cover new server costs come in naturally through increased exposure to a semi-untapped western comic user base.

I do think discussion on implementation would be healthy but in the end drawing more western comics fans into the kavita+ ecosystem would help defer costs and we have a passionate community who would likely be willing to help with maintenance/support through discord.

### Idea Category

Feature Enhancement

### Duration of Using Kavita

1+ year

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
Good news, Comic book metadata (series and issue level) along with ratings (critic and users) and reviews is now possible through Kavita+. 

https://github.com/Kareadita/Kavita/pull/3778

**Comment by majora2007** [comment]
This is delivered with v0.8.7.
